bin/generators/gateway/create.js

```diff
@@ -128,7 +128,8 @@ module.exports = class extends eg.Generator {
   }
 
   install () {
-    this.npmInstall(['express-gateway'], { save: true });
+    const { version: installedVersion } = require('../../../package.json');
```

Comment:
How did you come up with this path? Did you test it?

Reply:
@XVincentX   Yes, I have tested it. So I thought of picking the version from the package.json and pass it to express gateway so that the same version gets installed when generating the new instance of Express Gateway.

Reply:
Right! My only concern is how to make sure that the path is effectively correct — I'll pull down the branch and check it out.